pkgs/misc/emulators/wine/base.nix

```diff
@@ -22,10 +22,16 @@ stdenv.mkDerivation ((lib.optionalAttrs (! isNull buildScript) {
   buildInputs = toBuildInputs pkgArches (pkgs: (with pkgs; [
```

Comment:
Nevermind; I've confused title of the PR with commit message in the GitHub's UI.